.travis.yml

```diff
@@ -104,6 +104,11 @@ jobs:
         COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb:api-sanity-checker
       before_cache:
         - if [[ -n "$TRAVIS_TAG" ]]; then cp build/pkg/deb/*.deb $CACHE_DIR/; fi
+    - <<: *linux-build
+      env:
+        DIST_VERSION=bionic COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb:api-sanity-checker
```

Comment:
Try to remove the `COMPILER_VERSION` env variable. Pretty sure `gcc 4.9` isn't the default installed on bionic.

Reply:
Yep, that's right. I'll give it a shot.